routes/file.js

```diff
@@ -0,0 +1,15 @@
+const router = require("express").Router();
+const saveFile = require("../services/MulterService");
```

Comment:
`saveImage` might be a better name. In the future we might have different types of media as well.

Reply:
I was thinking of making it generalized. I can change if you say so.